app/jobs/dispute_status_update_job.rb

```diff
@@ -0,0 +1,61 @@
+class DisputeStatusUpdateJob < Que::Job
+  def run
+    @backlog = { 'activated': 0, 'closed': 0, 'activate_fail': [], 'close_fail': [] }
+               .with_indifferent_access
+
+    close_disputes
+    activate_disputes
+
+    Rails.logger.info "DisputeStatusUpdateJob - All done. Closed #{@backlog['closed']} and " \
+    "activated #{@backlog['activated']} disputes."
+
+    show_failed_disputes unless @backlog['activate_fail'].empty? && @backlog['close_fail'].empty?
+  end
+
+  def close_disputes
+    disputes = Dispute.where(closed: false).where('expires_at < ?', Time.zone.today).all
+    Rails.logger.info "DisputeStatusUpdateJob - Found #{disputes.count} closable disputes"
+    disputes.each do |dispute|
+      process_dispute(dispute, closing: true)
+    end
+  end
+
+  def activate_disputes
+    disputes = Dispute.where(closed: false, starts_at: Time.zone.today).all
+    Rails.logger.info "DisputeStatusUpdateJob - Found #{disputes.count} activatable disputes"
+
+    disputes.each do |dispute|
+      process_dispute(dispute, closing: false)
+    end
+  end
+
+  def process_dispute(dispute, closing: false)
+    intent = closing ? 'close' : 'activate'
```

Comment:
Do you really want to set the value of 'intent' variable or just to check it? If latter, than '==', suppose.

Reply:
It was supposed to set worded value for `intent`, so it can be later used for wording / manipulating named hash values.